### ISSUE TITLE
Embind broken with closure=1 and DYNAMIC_EXECUTION=0

### ISSUE THREAD

**Issue by emaxx-google** [emscripten-core/emscripten#15585]
Repro:
C++:
```
#include <emscripten/bind.h>
void Foo(int x) {}
EMSCRIPTEN_BINDINGS(baz) {
  emscripten::function("Foo", &Foo);
}
```
build:
```
emcc --bind --closure=1 -O3 -s DYNAMIC_EXECUTION=0 test.cc -o test.html
```
run:
```
Module.Foo(1);
```
Fails with:
```
test.js:formatted:460 Uncaught TypeError: b[(q + 2)].F is not a function
    at Object.Foo (test.js:formatted:460)
    at <anonymous>:1:8
```

**Comment by kripken** [comment]
Thanks @emaxx-google , should be fixed in #15597

**Comment by TJ4Urservices** [comment]
Please
